Mathlib/RingTheory/Spectrum/Prime/ConstructibleSet.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2024 Yaël Dillies, Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Andrew Yang
+-/
+import Mathlib.Order.SuccPred.WithBot
+import Mathlib.RingTheory.Spectrum.Prime.Topology
+
+/-!
+# Constructible sets in the prime spectrum
+
+This file provides tooling for manipulating constructible sets in the prime spectrum of a ring.
+
+## TODO
+
+Link to constructible sets in a topological space.
+-/
+
+open Finset
+open scoped Polynomial
+
+namespace PrimeSpectrum
+variable {R S T : Type*} [CommRing R] [CommRing S] [CommRing T]
```

Comment:
It seems nothing in this file needs additive inverses yet; can you change to CommSemiring?